ui/bits/src/bits.relayTourForm.ts

```diff
@@ -0,0 +1,5 @@
+site.load.then(() => {
+  $('.form3[action="/broadcast/new"] #form3-info_timeZone').each(function (this: HTMLSelectElement) {
+    if (!$('.is-invalid').length) this.value = Intl.DateTimeFormat().resolvedOptions().timeZone;
```

Comment:
It was the best way I found to prevent changes if there is an error or if it's in the edit form.

Reply:
if this is specific to the 4 best players, i don't understand an assumption they would all be in the same time zone and i don't understand why we would want to default it to the timezone of the person filling this form.

but if this is what you want to do, lgtm